tools/replay.cpp

```diff
@@ -34,6 +34,9 @@ int main(int argc, char* argv[])
      , "Input file created by Umpire library with UMPIRE_REPLAY=On"
      , cxxopts::value<std::string>(), "FILE"
     )
+    (  "l, largefile"
```

Comment:
I don't think this is a good name for the option - it should be more descriptive of what the option does, not _why_ it does it.